src/custom/whatsapp.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g stroke="currentColor" clip-path="url(#a)"><path d="m7.54239 19.4858-3.8416.9202.56195-4.2954c-1.14134-1.9873-1.56086-4.3844-.99868-6.77745C4.44985 4.28538 9.5543 1.16665 14.6652 2.36727c5.1108 1.20062 8.2927 6.26594 7.1069 11.31373-1.1858 5.0477-6.2902 8.1665-11.4011 6.9658-1.02742-.2413-1.9769-.6389-2.82861-1.161Z"/><path d="m12.9801 14.4579 1.3379-1.5365 2.4896 1.4924-.496.5696c-.3303.3795-.8422.5471-1.333.4367-1.5005-.3376-2.8768-1.0516-4.1289-2.1418-1.25209-1.0903-2.14852-2.3553-2.6893-3.79509-.17689-.47096-.08123-1.00099.24914-1.3804l.49598-.56961 1.82068 2.26067-1.33794 1.53653"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h24v24H0z"/></clipPath></defs></svg>
```

Comment:
We're in minor optimisation land, but it would be good to pop this through SVGOMG to reduce the path precision https://jakearchibald.github.io/svgomg/

You can't preview it properly there as we rely on attributes passed by `<Icon />`, but you should be able to test them in the sandbox to check that the optimisation works well.

Reply:
Miguel has done it with all the icons before submitting them to us

Reply:
We definitely don't need a precision of 5 decimal places, we can probably get it down to 1. Can you put them through and compare?

Reply:
done